src/extension/background-script/connectors/lnd.js

```diff
@@ -61,6 +61,12 @@ class Lnd extends Base {
     });
   };
 
+  getTransactions = () => {
+    return this.request("GET", "/v1/transactions", undefined, {
```

Comment:
oh and this should be "list payments"
https://api.lightning.community/#v1-payments
https://api.lightning.community/#listpayments

The payments are the lightning payments. Transactions are the on-chain bitcoin transactions.

Reply:
thanks for making that clear - was confused already that they seemed not up to date

Reply:
https://github.com/bumi/lightning-browser-extension/issues/45